contracts/eip20/EIP20Factory.sol

```diff
@@ -37,7 +37,7 @@ contract EIP20Factory {
         public
     returns (address) {
 
-        EIP20 newToken = (new EIP20(_initialAmount, _name, _decimals, _symbol));
+        SafeEIP20 newToken = (new SafeEIP20(_initialAmount, _name, _decimals, _symbol));
```

Comment:
Do we want to undo this change?

Reply:
@skmgoldin I had the impression that we were going to revert the factory back to EIP20.